nibe/data/s735.csv

```diff
@@ -455,10 +458,6 @@ Date, periodic hot water	MODBUS_INPUT_REGISTER	1561	1		-	-	-	-
 Blocked compressors	MODBUS_INPUT_REGISTER	2174	1		6	0	0	0
 Used compressors hot water	MODBUS_INPUT_REGISTER	1571	1		4	0	0	0
 Used compressors heating	MODBUS_INPUT_REGISTER	1572	1		4	0	0	0
-Hot water, including int. add. heat	MODBUS_INPUT_REGISTER	1575	10	kWh	6	0	9999999	0
-Heating, including int. add. heat	MODBUS_INPUT_REGISTER	1577	10	kWh	6	0	9999999	0
-Hot water, compressor only	MODBUS_INPUT_REGISTER	1583	10	kWh	6	0	9999999	0
-Heating, compressor only	MODBUS_INPUT_REGISTER	1585	10	kWh	6	0	9999999	0
```

Comment:
I think these might enable based features added to the pump. If a flow meter is added, these become active. So likely should not be removed.

Reply:
These are not removed. I just moved them to the location where they were in the new CSV.

Reply:
Oh right. Weird i thought our json was sorted on ID